Sources/Extensions/SwiftStdlib/StringExtensions.swift

```diff
@@ -678,6 +678,34 @@ public extension String {
     }
     #endif
 
+    /// SwifterSwift: Check if string contains substring exactly `times` times
+    ///
+    ///     "hellohello".contains("hello", times: 2) -> true
+    ///     "hello".contains("h", times: 2) -> false
+    ///     "hello.contains("l", times: 2) -> true
```

Comment:
Looks like you missed a quote for the string value.

Reply:
OOPS! Let me add that in! Thanks 👍